WcaOnRails/app/models/registration.rb

```diff
@@ -192,4 +192,30 @@ def events_to_associated_events(events)
       registration_competition_events.find_by_competition_event_id(competition_event.id) || registration_competition_events.build(competition_event: competition_event)
     end
   end
+
+  def serializable_hash(options = nil)
+    status = if accepted?
+               'a'
+             elsif pending?
+               'p'
+             else
+               'd'
+             end
+    json = {
+      class: self.class.to_s.downcase,
+      id: id,
+      competition_id: competition_id,
+      user: user.as_json(options),
```

Comment:
I'm going to go with exporting the user in /registrations rather than just user_id. This should be the default and expected behavior (/competitions works the same way).